packages/angular-cli/models/config/config.ts

```diff
@@ -80,10 +80,16 @@ export class CliConfig<JsonType> {
 
     try {
       content = JSON.parse(configContent);
+    } catch (err) {
+      throw 'Parsing angular-cli.json failed. Please make sure your angular-cli.json is valid'
```

Comment:
Using the class currently declared as `InvalidConfigError` always just prints "Error", no matter what you pass to it. For that reason I fixed it to use the way provided in [this SO answer](http://stackoverflow.com/a/32750746/1744702). Now it prints the error correctly. Also applied the same for another existing use of the `InvalidConfigError` class.